node/cn/handler_test.go

```diff
@@ -380,6 +380,105 @@ func TestBroadcastTxsFrom_DefaultCase(t *testing.T) {
 	pm.BroadcastTxs(txs)
 }
 
+func TestReBroadcastTxs_CN(t *testing.T) {
```

Comment:
Can this test case be failed?

Reply:
Yes, inside `ReBroadcastTxs`, if any peers are selected, `peer.SendTransactions` will be called and this will make the test failed. But when it is CN, it does not do anything but return.